npm/grep/package.json

```diff
@@ -13,6 +14,7 @@
     "globby": "^11.0.4"
   },
   "devDependencies": {
+    "cypress": "^11.2.0",
```

Comment:
Do we want this using the dev version of Cypress - current within the repo? `0.0.0-development` I can't imagine someones going to maintain updating this in the future.

Reply:
The issue is that we are using `cypress-expect` which doesn't work with our binary 👎 . And we need to be able to validate the output with the expected results objected in the `/expected/my-expected-results.json`

Reply:
How come that doesn't work with our binary, curious?

Reply:
It's because `cypress-expect` wraps around the module API